src/ipfs.ts

```diff
@@ -96,6 +96,9 @@ export async function handleStrategiesParsedMetadata(metadataUri: string) {
       strategiesParsedMetadataItem.symbol = metadata.properties.symbol;
     }
     if (metadata.properties.token) strategiesParsedMetadataItem.token = metadata.properties.token;
+    if (metadata.properties.payload) {
+      strategiesParsedMetadataItem.payload = metadata.properties.payload;
```

Comment:
Currently it's stringified tree, but it will likely be replaced by `ipfs://` URL so it's not stored by the API in case tree is long.